tests/test_radiation.py

```diff
@@ -0,0 +1,207 @@
+"""Test module for abiotic.radiation.py."""
+
+from contextlib import nullcontext as does_not_raise
+
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize(
+    argnames=["cname"],
+    argvalues=[
+        ("BEER_REGRESSION",),
+        ("BOLZMAN_CONSTANT",),
+        ("CANOPY_EMISSIVITY",),
+        ("CELSIUS_TO_KELVIN",),
+        ("CLOUDY_TRANSMISSIVITY",),
+        ("FLUX_TO_ENERGY",),
+        ("SECOND_TO_DAY",),
+        ("SOIL_EMISSIVITY",),
+        ("TRANSMISSIVITY_COEFFICIENT",),
+    ],
+)
+def test_import_constants(cname):
+    """Test constants can be imported."""
+
+    # Get the module that should contain the constants
+    import virtual_rainforest.models.abiotic.radiation as rad
+
+    # Check that the constant can be retrieved from the module without
+    # raising an AttributeError.
+    with does_not_raise():
+        assert getattr(rad, cname)
```

Comment:
As I think I said on the last version of this PR, I don't think you need to test that the constants are present. Just test whether your functions work.